COMMANDS.md

```diff
@@ -3,6 +3,22 @@
 This is a list of commands currently implemented in Hoosky Bot.
 
 > Note: `[]` indicates a variable argument.
+> Note: `[?]` indicates an optional argument.
+
+## `/birthday` [WIP]
+
+| Command                                  | Description                                                          |
```

Comment:
Totally not something with this PR but we might want to make a separate column in the commands listing for the permissions required, just so it doesn't clutter up the description but also makes it easier to find.

Reply:
I was thinking about that as well ~ will update 👍